mangaki/mangaki/factories.py

```diff
@@ -21,7 +21,6 @@ class Meta:
 
     username = factory.Faker('user_name')
     email = factory.LazyAttribute(lambda o: '{}@mangaki.fr'.format(o.username))
-    profile = factory.RelatedFactory(ProfileFactory, 'user')
```

Comment:
Thanks to the new signal, a new `Profile` will be created automatically for the user.